python/ray/tests/test_actor_bounded_threads.py

```diff
@@ -1,6 +1,7 @@
 import sys
 import os
 import psutil
+import pytest
```

Comment:
@rynewang lint is failing btw https://buildkite.com/ray-project/premerge/builds/5638#018a858b-a713-4e4a-9974-16749141ead0/140-381

Reply:
fixed